pkg/services/featuremgmt/toggles_gen.go

```diff
@@ -308,7 +308,7 @@ const (
 	FlagAdvancedDataSourcePicker = "advancedDataSourcePicker"
 
 	// FlagFaroDatasourceSelector
-	// Enable the data source selector within the Frontend Apps section of the Frontend Observability
+	// Enable the data source selector within the Frontend Apps section of the Frontend Observability 
```

Comment:
could we do it the other way - change the inputs so we don't have a trailing space? i've had linters complain